### DISCUSSION TITLE
Leave Insert Mode after inactivity

### DISCUSSION THREAD

**Discussion by i-zanis** [JetBrains/ideavim#466] [Ideas]
Hello Alex and IdeaVim Team,

Thank you from the bottom of my heart for the contribution to this project. 

I would like to ask whether there is a possibility to implement a "Leave Insert Mode after inactivity" feature as there is limited support for VimScript.

Many thanks,

**Comment by AlexPl292** [comment]
Hi there. Thank you for your kind words!
Do you use this feature in vim? What approach do you use to achieve that?
